INTEGRATION.md

```diff
@@ -0,0 +1,166 @@
+# Gateway Integration
+
+![wip](https://img.shields.io/badge/status-wip-orange.svg?style=flat-square)
+
+**Author(s)**:
+- [Mark Gaiser](https://github.com/markg85/)
+
+**Maintainer(s)**:
+- [Mark Gaiser](https://github.com/markg85/)
+
+* * *
+
+## Summary
+
+Defines the decision model an IPFS enabled application should use to find a IPFS gateway. Simultaneously defines how an IPFS implementation should expose it's gateway.
+
```

Comment:
s/IPFS gateway/HTTP gateways/ (note the plural)

this change throughout - should be about configuring a gateway list.